RELEASING.md

```diff
@@ -93,128 +70,101 @@ green, you may create a release as follows:
+
+1. Update/Install releasetool with the following:
 
     ```sh
-    $ git commit -am "Release <gem> <version> ..."
+    $ python3 -m pip install --user --upgrade gcp-releasetool
     ```
 
-1. Verify that your changes are complete, and that the version numbers all
-   match.
+1. While on the master branch, cd into the directory of the gem you would like to release, and run:
 
     ```sh
-    $ git show
+    $ python3 -m releasetool start
     ```
 
-1. Repeat steps 4 through 13 if you are releasing multiple gems.
+    1. If it's your first time running this, releasetool will likely ask you for a GitHub API token with write:repo_hook and public_repo permissions from https://github.com/settings/tokens.
+
+    1. Next, releasetool will present you with an automatically generated changelog. Use the information from steps 4 and 5, as well as the provided PR numbers to edit and complete the changelog. Remove the "(#111)" style PR references before continuing.
+
+    1. Releasetool will then ask you if your change is considered major, minor, or patch. This project uses [semantic versioning](http://semver.org).
+
+    1. Releasetool will then create and switch to a branch called "release-#{gem-name}-v#{version}", make all the necessary changes to the changelog and gem version, and open a [pull request](https://github.com/googleapis/google-cloud-ruby/pulls).
+
+1. Review the PR created by releasetool in the previous step, apply the appropriate label (i.e. "api: #{gem_name}"), and request a review from googleapis/yoshi-ruby.
+
+1. Repeat steps 3 through 12 if you are releasing multiple gems.
 
 1. If you updated `google-cloud/google-cloud.gemspec` for a version change to
-   any gem, repeat steps 5 through 13 for the `google-cloud` gem.
+   any gem, repeat steps 4 through 12 for the `google-cloud` gem.
 
 1. If you updated `stackdriver/stackdriver.gemspec` for a version change to any
-   gem, repeat steps 5 through 13 for the `stackdriver` gem.
+   gem, repeat steps 4 through 12 for the `stackdriver` gem.
 
-1. If any dependencies have been updated in the previous steps, test that all
-   version dependencies are correct.
+1. After your pull request has passed all checks and been approved by reviewers,
+   **Squash and merge** it. This will trigger a build on [Kokoro](#Checking-the-status-of-Kokoro-builds), which will create the [release](https://github.com/googleapis/google-cloud-ruby/releases) and build and push the gem to [rubygems](https://rubygems.org/). If yoshi-automation reports a failure creating the [release](https://github.com/googleapis/google-cloud-ruby/releases), file a bug on [releasetool's issue tracker](https://github.com/googleapis/releasetool/issues), and [write the release manually](writing-a-release-manually). If yoshi-automation reports that the release build has failed, follow [these steps](#Checking-the-status-of-Kokoro-builds), selecting the Kokoro build titled "Kokoro - Release".
```

Comment:
Does the GitHub release actually publish the gem to Rubygems? If so, this is OK, but otherwise maybe change "creating the [release]" -> "creating the [GitHub Release post]" (or similar)